en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,453 @@
+# Redis Connector Example
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs and geospatial indexes with radius queries.
+
+## What you'll build
+
+Given below is a sample scenario that demonstrates how to work with the WSO2 EI Redis Connector and access the Redis server, using Redis commands.
+
+The user sends the request for invoke to the created API to get IBM stock details. These REST call will get converted into SOAP calls and sent to the back-end service. The response will be sent to the client and in will use to create a `Redis list` to store IBM stock volume details inside the Redis server.
```

Comment:
for invoke to the created API --> to an API

Reply:
These REST call --> this REST call

Reply:
SOAP calls --> a SOAP message

Reply:
The response will be sent to the client and in will use to create a `Redis list` to store IBM stock volume details inside the Redis server.

I believe what you want to express is : 

While the response from the backend service is converted back to json and sent back to the API caller, WSO2 EI will extract `stock volume details` from the response and use xxx operation (bold operation name) to store it into configured Redis server.

Reply:
Updated as suggested one.